apps/challenges/challenge_config_utils.py

```diff
@@ -545,7 +574,7 @@ def validate_challenge_config_util(
                         data["id"], missing_keys_string
                     )
                     error_messages.append(message)
-        if isfile(test_annotation_file_path):
+        if test_annotation_file_path is not None and isfile(test_annotation_file_path):
```

Comment:
What would happen if `test_annotation_file_path` is None?

Reply:
@AyushR1 Yes, that was the case when we don't have the check. I am just asking what will happen with the new code?